images/docker-stacks-foundation/run-hooks.sh

```diff
@@ -1,46 +1,6 @@
 #!/bin/bash
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
+# echo "WARNING: Use run-hooks.py instead"
```

Comment:
Unfortunately, we have to comment this line, otherwise, tests fail, because they do not expect warnings.